docs/panel/content-objects.md

```diff
@@ -14,65 +16,322 @@ or entry in the left menu:
 
 ![](images/ContentMenu.png){: .center .border}
 
-To quickly browse and search through large amounts of data, the content browser provides a convenient grid interface. Users can customize the grid according to their preferences:
+The content browser provides a convenient grid interface to browse and search through large amounts of data quickly.
+Users can customize the grid according to their preferences:
 
 ![Flotiq content browser](images/GridView.png){: .center .width75 .border}
 
 From the grid, the users can go to the form that allows them to edit particular Content Objects:
 
 ![Flotiq Object Editor](images/EditObject.png){: .center .width75 .border}
 
-The form view also displays information about previous versions of a particular Content Object. From this view, you can also switch to older revisions. To save older revision, click on the version tile, and then click the 'Save' button.
+The form view also displays information about previous versions of a particular Content Object.
```

Comment:
Can we move the Content Versioning description to a separate section (like in `Deleting content`)? It can improve readability and increase SEO for the "content versioning" phrase.

Reply:
moved to Content versioning section